src/debug/ee/arm64/arm64walker.cpp

```diff
@@ -125,8 +123,9 @@ BYTE*  NativeWalker::SetupOrSimulateInstructionForPatchSkip(T_CONTEXT * context,
 
        
     }
-    else if ((opcode & 0xFF000000) == 0x58000000) //Load register (literal)
-    {
+    
+    else if ((opcode & 0x3BF000000) == 0x18000000) //Load register (literal)
+     {
         
         offset = (opcode & 0x1FFFFFE0) << 2;
```

Comment:
this is incorrect is should be:

INT32 disp = (opcode << 8);   // shift the sign bit into bit 31
          disp = (disp >> 13);      // shift the imm19, such that the original bit 5 is now bit 0
          disp = (disp << 2);        // shift in the two zero bits



((opcode >> 5) && 0x7FFFF) << 2;   // Extract 19-bit immediate value